downlords-faf-client.install4j

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<install4j version="7.0.7" transformSequenceNumber="7">
-  <directoryPresets config="./src/media/appicon" />
+<install4j version="7.0.6" transformSequenceNumber="7">
```

Comment:
We should rather go forward than backward ;-)

Reply:
I think the entire install4j file can be reverted